src/validateArchive.js

```diff
@@ -9,7 +9,7 @@ export default function validateArchive(totalBytes, entries) {
   ];
   const fileSizes = entries.map((entry) => ({
     name: entry.name,
-    size: entry.stats ? entry.stats.size : (entry.size || 0),
+    size: entry.stats ? entry.stats.size : entry.size || 0,
```

Comment:
Not important but this change seemed to make things harder to read.

Reply:
Yeah :/